.eslintrc

```diff
@@ -12,7 +12,7 @@
     "react/prop-types": "off",
     "react/no-unescaped-entities": "off",
     "no-unused-vars": "warn",
-    "unused-imports/no-unused-imports": "error",
+    "unused-imports/no-unused-imports": "warn",
```

Comment:
🤔

Reply:
Should I leave it at error? I don't want to be too agressive.

Reply:
IMO it is aggressive, but it helps us to make the app size perfect without any extra imports 🙇🏻